corehq/apps/linked_domain/remote_accessors.py

```diff
@@ -57,22 +57,24 @@ def _convert_app_from_remote_linking_source(app_json):
     return app
 
 
-def pull_missing_multimedia_for_app(app):
-    missing_media = _get_missing_multimedia(app)
+def pull_missing_multimedia_for_app(app, old_multimedia_ids=None):
```

Comment:
Similar question as https://github.com/dimagi/commcare-hq/pull/25010/files#r310487724

Reply:
Where would you move this one exactly? It's already in `linked_domain`. I'm not in favor of making a new `linked_app` app.

Reply:
@orangejenny 
`linked_domain/views/utils` or `linked_domain/utils`.

hmm, I was thinking `linked_app` is a separate entity from `linked_domain` and should have its own app? thoughts?

Reply:
https://github.com/dimagi/commcare-hq/pull/25010/commits/1ce375dc4af17100c6d94109ed366d2aacaa0743